src/main/java/com/github/tommyettinger/digital/MathTools.java

```diff
@@ -53,6 +53,18 @@ private MathTools() {
      * false-positive equivalence with very small inputs.
      */
     public static final float FLOAT_ROUNDING_ERROR = 0x1p-20f; // was 0.000001f
+    
+    /**
+     * The smallest measurable difference between a given {@code float} value and a directly adjacent value.
+     * Useful for converting a 64-bit {@code long} value to a gradient between 0 and 1.
+     */
+    public static final float EPSILON = 0x1p-24f;
+    
+    /**
+     * The smallest measurable difference between a given {@code double} value and a directly adjacent value.
```

Comment:
Also the comment about an ulp above applies here. This might be the smallest non-zero distance possible between two doubles returned by `Random#nextDouble()`, but Random specifically might not be so precise. All of the EnhancedRandom classes in juniper have a nextDouble() that matches this precision.